README.adoc

```diff
@@ -1,6 +1,6 @@
 = google-java-format-gradle-plugin
 :release-version: 0.8
-:default-google-java-format-version: 1.6
+:default-google-java-format-version: 1.7
```

Comment:
This variable belongs to the latest released version and not to the current snapshot version, so it should still be `1.6`

Reply:
Reverted